tool/sync/dependencies.py

```diff
@@ -0,0 +1,142 @@
+# published by the Free Software Foundation, either version 3 of the
+# License, or (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with this program.  If not, see <https://www.gnu.org/licenses/>.
+#
+
+"""
+sync-dependencies.py updates bazel dependencies between @vaticle repositories
+
+Example usage:
+bazel run @vaticle_dependencies//tool/sync:dependencies -- --source typedb-driver@1a2b3c4d1a2b3c4d1a2b3c4d1a2b3c4d1a2b3c4g
+"""
+
+import argparse
+import tool.common.common as tc
+import github
+import hashlib
+import hmac
+import json
+import os
+import re
+import subprocess as sp
+import sys
+
+IS_CIRCLECI = bool(os.getenv('CIRCLECI'))
+IS_FACTORY = bool(os.getenv('FACTORY_REPO'))
+IS_CI_ENV = IS_CIRCLECI or IS_FACTORY
+
+GITHUB_TOKEN = os.getenv('SYNC_DEPENDENCIES_TOKEN')
+if GITHUB_TOKEN is None:
+    raise Exception("$SYNC_DEPENDENCIES_TOKEN is not set!")
+
+BOT_HOST = 'https://bot.vaticle.com'
+if not IS_CI_ENV:
+    BOT_HOST = 'http://localhost:8000'
+
+BOT_SYNC_DEPS = '{0}/sync/dependencies'.format(BOT_HOST)
+
+CMDLINE_PARSER = argparse.ArgumentParser(description='Automatic updater for Vaticle inter-repository dependencies')
+CMDLINE_PARSER.add_argument('--dry-run', help='Do not perform any real actions')
+CMDLINE_PARSER.add_argument('--source', required=True)
+
+COMMIT_SUBJECT_PREFIX = "//tool/sync:dependencies"
+regex_git_commit = r'[0-9a-f]{40}'
+regex_git_tag = r'([0-9]+\.[0-9]+\.[0-9]+)'
+
+vaticle = 'vaticle'
+github_connection = github.Github(GITHUB_TOKEN)
+github_org = github_connection.get_organization(vaticle)
+
+
+def is_building_upstream():
+    """ Returns False is running in a forked repo"""
+    if IS_CIRCLECI:
+        return vaticle in os.getenv('CIRCLE_REPOSITORY_URL', '')
+    elif IS_FACTORY:
+        return vaticle == os.getenv('FACTORY_OWNER')
+    else:
+        return False
+
+
+def exception_handler(fun):
+    """ Decorator printing additional message on CalledProcessError """
+
+    def wrapper(*args, **kwargs):
+        # pylint: disable=missing-docstring
+        try:
+            fun(*args, **kwargs)
+        except sp.CalledProcessError as ex:
+            print(f'An error occurred when running {ex.cmd}. Process exited with code {ex.returncode} and message {ex.output}')
+            print()
+            raise ex
+
+    return wrapper
+
+
+def short_commit(commit_sha):
+    return sp.check_output(['git', 'rev-parse', '--short=7', commit_sha], cwd=os.getenv("BUILD_WORKSPACE_DIRECTORY")).decode().replace('\n', '')
+
+
+@exception_handler
+def main():
+    if not is_building_upstream():
+        print('//ci:sync-dependencies aborted: not building the upstream repo on @vaticle')
```

Comment:
Just use `Sync dependencies aborted` - the Bazel path is unlikely to be the same like this forever/even be this now!